Package.swift

```diff
@@ -30,30 +33,106 @@ let package = Package(
         .package(
             name: "Quick",
             url: "https://github.com/Quick/Quick.git", from: "3.0.0"),
-    ],
+    ] + environment.dependencies,
     targets: [
-        .binaryTarget(
-            name: "MaasCore",
-            // Used for production
-//            url: "https://github.com/trafi/maas-components/releases/download/0.1.0-dev05/MaasCore.xcframework.zip",
-//            checksum: "d0e9dfa86527369490cb3293e6370e3d8844d14cd35a19bde44433fc337921ed"),
-            // Used for development
-            path: "common/core/build/bin/xcframework/MaasCore.xcframework"),
-
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages this package depends on.
         .target(
-            name: "MaaS",
-            dependencies: ["MaasCore", "Swappable", "MaasTheme"],
-            path: "ios/Sources/MaaS"),
-
+            name: "Maas",
+            dependencies: ["Swappable"],
+            path: "ios/Sources/Maas"),
+        .target(
+            name: "Examples",
```

Comment:
I'm not sure what's clearer to people, and it really doesn't matter much in the grand scheme of things, but ["samples" (vs "examples")](https://english.stackexchange.com/questions/390357/whats-the-difference-between-example-and-sample) is more common in software as far as I know. Do Apple / popular iOS frameworks more commonly use the term "sample" or "example"?

Reply:
Firebase - example, Facebook - sample. I think this doesn't matter. 🙈

Reply:
OK, agree 🙂